easybuild/easyconfigs/b/Bonito/Bonito-0.3.2-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,127 @@
+easyblock = 'PythonBundle'
+
+name = 'Bonito'
+version = '0.3.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/nanoporetech/bonito'
+description = "Convolution Basecaller for Oxford Nanopore Reads"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('PyTorch', '1.3.1', versionsuffix),
+    ('h5py', '2.10.0', versionsuffix),
+    ('Mako', '1.1.0'),
+    ('PyYAML', '5.1.2'),
+    ('apex', '20200325', versionsuffix),
+    ('minimap2', '2.17'),
+    ('CuPy', '8.2.0', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('toml', '0.10.2', {
+        'checksums': ['b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f'],
+    }),
+    ('tqdm', '4.54.1', {
```

Comment:
Unless this requires the newer version, we should use `tqdm-4.41.1-GCCcore-8.3.0.eb` instead.